[dp-thread-pool] Installed to wrong path

### Is your feature request related to a problem? Please describe.

`vcpkg install dp-thread-pool` installed dp-thread-pool to `%VCPKG_ROOT%\installed\x64-windows\include\thread-pool-0.7.0\thread_pool`, which has an unnecessary `thread-pool-0.7.0`

If `#include <thread-pool-0.7.0/thread_pool/thread_pool.h>`,will result in a compilation error:

```
installed\x64-windows\include\thread-pool-0.7.0\thread_pool\thread_pool.h(18,10): error C1083: Cannot open include file: 'thread_pool/thread_safe_queue.h': No such file or directory
```

### Proposed solution

Remove `thread-pool-0.7.0`

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

--- a/ports/dp-thread-pool/include.diff
+++ b/ports/dp-thread-pool/include.diff
@@ -1,0 +1,13 @@
+diff --git a/include/thread_pool/thread_pool.h b/include/thread_pool/thread_pool.h
+index 4dbe54f..4b97ec5 100644
+--- a/include/thread_pool/thread_pool.h
++++ b/include/thread_pool/thread_pool.h
+@@ -15,7 +15,7 @@
+ #    endif
+ #endif
+ 
+-#include "thread_pool/thread_safe_queue.h"
++#include "thread_safe_queue.h"
+ 
+ namespace dp {
+     namespace details {

--- a/ports/dp-thread-pool/portfile.cmake
+++ b/ports/dp-thread-pool/portfile.cmake
@@ -6,6 +6,8 @@ vcpkg_from_github(
     REF ${VERSION}
     SHA512 c2a75117f7def0dacc2679f8eb70835acfba58d1aba7beec9bf0a29cdb23883222294dc4b04b77e323f8ecb8623b70d728bee46bf2c5a4fd6711a749c9709981
     HEAD_REF master
+    PATCHES
+        include.diff
 )
 
 vcpkg_cmake_configure(

--- a/ports/dp-thread-pool/vcpkg.json
+++ b/ports/dp-thread-pool/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "dp-thread-pool",
   "version": "0.7.0",
+  "port-version": 1,
   "maintainers": "Paul Tsouchlos <developer.paul.123@gmail.com>",
   "description": "dp::thread_pool - fast, lightweight C++20 thread pool.",
   "homepage": "https://github.com/DeveloperPaul123/thread-pool",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2486,7 +2486,7 @@
     },
     "dp-thread-pool": {
       "baseline": "0.7.0",
-      "port-version": 0
+      "port-version": 1
     },
     "dpdk": {
       "baseline": "24.07",

--- a/versions/d-/dp-thread-pool.json
+++ b/versions/d-/dp-thread-pool.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "bc8e4a89a343d5cadc4838943f5668551543758b",
+      "version": "0.7.0",
+      "port-version": 1
+    },
+    {
       "git-tree": "9016b440875e13fa99e727ed07d87eb06c6ccee6",
       "version": "0.7.0",
       "port-version": 0

